src/navigation/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon, IconTypes } from '../../dataDisplay/Icon';
+import Text from '../../dataDisplay/Text';
+import Link from '../../inputs/Link';
+import { ThemeColors } from '../../theme';
+
+const Wrapper = styled.div`
+  padding: 16px 0;
+  box-sizing: border-box;
+  max-height: 54px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const ElementWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin: auto 5px;
+`;
+
+const StyledLink = styled(Link)`
+  font-family: ${({ theme }) => theme.fonts.fontFamily};
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 800;
+  cursor: text;
+  margin: auto 5px;
+`;
+
+const StyledText = styled(Text)`
+  font-family: ${({ theme }) => theme.fonts.fontFamily};
+  text-transform: uppercase;
+  font-weight: normal;
+  margin: auto 5px;
+`;
+
+type BreadcrumbElementProps = {
+  iconType?: IconTypes;
+  text: string;
+  counter?: string;
+  color?: ThemeColors;
+};
+
+export const BreadcrumbElement = ({
+  iconType,
+  text,
+  counter,
+  color = 'primary',
+}: BreadcrumbElementProps): React.ReactElement => {
+  return (
+    <ElementWrapper>
+      {iconType && <Icon type={iconType} size="sm" color={color} />}
+      {color === 'primary' ? (
+        <StyledLink size="md" color={color}>
+          {text}
+        </StyledLink>
+      ) : (
+        <StyledText size="md" color={color}>
+          {text}
+        </StyledText>
+      )}
```

Comment:
If it would be moved to the component body like `let TextComponent = ...` I'd find it more readable personally. Not a blocker tho

Reply:
I extracted this logic to a `BreadcrumbText` component